rfcs/models/infrastructure/relays.md

```diff
@@ -73,11 +72,32 @@ Once a driver is designed for a given DLT, it can be reused in any network built
 
 ## Relay Functional API
 
-_TBD: List API function specs exposed to application clients, relays, and drivers respectively. Message formats will be described in 'formats/communication'_
+### API for Application Client
+
+- **RequestState(networks.networks.NetworkQuery): returns common.ack.Ack**
+    This endpoint is for the client application to request remote state via local relay. Since this request is asynchronous w.r.t the network, the request info/state machine is stored in a database on the requesting relay indexed via `request_id`. This `request_id` can be used to poll for status and the response using `GetState` endpoint. The local relay then sends the Query to the remote relay via an API available for relay-relay communication (explained below). This API takes a message of type [NetworkQuery](../../formats/communication/relay.md#networkquery) as argument, and returns [Ack](../../formats/views/request.md#ack) message.
+    
+- **GetState(networks.networks.GetStateMessage): returns common.state.RequestState**
+    This RPC endpoint is used by client application for polling the local relay to get the response to the query by using `request_id` returned in previous API. The local relay fetches the database state based on provided `request_id` and prepares the response. It takes a message of type [GetStateMessage](../../formats/communication/relay.md#getstatemessage) as argument, and returns [RequestState](../../formats/views/request.md#requeststate) message.
```

Comment:
- All of these are RPC endpoints, right? To be precise, gRPC? And not just this particular function? If so, just state that in the intro, before the list, saying that these are RPC endpoints that can be implemented directly a well-known protocol like gRPC.
- Instead of using "previous API", just specify the name (in this case `RequestState`)